windows/MAVProxyWinBuild.bat

```diff
@@ -1,11 +1,11 @@
 rem build the standalone MAVProxy.exe for Windows.
-rem This assumes Python is installed in C:\Python36
+rem This assumes Python is installed in C:\Program Files (x86)\Python37-32

```

Comment:
This is odd considering the 3.8 version mentioned earlier.

We're testing against a version we're not requiring in the intall?

Reply:
Fixed - now just using the user's default python version.
Note this script is not used in the appveyor CI build. This is primarily for developers to build locally.